### PR TITLE
Testing: git push auto fixed files

### PR DESCRIPTION
**Pull request by lukelloyd1985**
Following on from PR #4176 to fix crash
Fixes `apply_fixes` typo

## Proposed Changes

Testing git add, commit & push when there are fixed sources files

### Author/Contributor
- [ ] Add entry to the [CHANGELOG](https://github.com/oxsecurity/megalinter/blob/main/CHANGELOG.md) listing the change and linking to the corresponding issue (if appropriate)
- [ ] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

**Comment by nvuillam** [comment]
@lukelloyd1985 plz solve the error in this PR MegaLinter job, i think there is a syntax error :)

**Comment by nvuillam** [comment]
```
    File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
    File "/github/workspace/megalinter/reporters/UpdatedSourcesReporter.py", line 77
      "[Updated Sources Reporter] Failed to git push auto fixes: " + str(exp.message)
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  SyntaxError: invalid syntax. Perhaps you forgot a comma?
```

File: megalinter/reporters/UpdatedSourcesReporter.py [hunk 1]
```diff
@@ -76,6 +75,7 @@ def produce_report(self):
                     except Exception as exp:
                         logging.error(
                             "[Updated Sources Reporter] Failed to git push auto fixes: " + str(exp.message)
```
Comment (nvuillam) [inline megalinter/reporters/UpdatedSourcesReporter.py:77]: @lukelloyd1985 missing `,` at the end of this line :)

Comment (lukelloyd1985) [reply megalinter/reporters/UpdatedSourcesReporter.py]: dammit :(

**Pull request merged**